packages/wrangler/src/metrics/metrics-config.ts

```diff
@@ -63,13 +55,11 @@ export interface MetricsConfig {
  * If the current process is not interactive then we will cannot prompt the user and just assume
  * we cannot send metrics if there is no cached or project-level preference available.
  */
```

Comment:
This comment could do with updating?

Reply:
5fcef96670a2fa2e897f868dbce666a7efbe7b61